src/main/java/org/folio/service/loader/RecordLoaderServiceImpl.java

```diff
@@ -20,30 +20,31 @@
 @Service
 public class RecordLoaderServiceImpl implements RecordLoaderService {
 
-  private SourceRecordStorageClient client;
+  private StorageClient client;
 
-  public RecordLoaderServiceImpl(@Autowired SourceRecordStorageClient client) {
+  public RecordLoaderServiceImpl(@Autowired StorageClient client) {
     this.client = client;
   }
 
   @Override
-  public SrsLoadResult loadMarcRecordsBlocking(List<String> uuids, OkapiConnectionParams okapiConnectionParams) {
-    Optional<JsonObject> optionalRecords = client.getByIds(uuids, okapiConnectionParams);
+  public SrsLoadResult loadMarcRecordsBlocking(List<String> uuids, OkapiConnectionParams okapiConnectionParams, int partitionSize) {
+    Optional<JsonObject> optionalRecords = client.getByIdsFromSRS(uuids, okapiConnectionParams, partitionSize);
     SrsLoadResult srsLoadResult = new SrsLoadResult();
     if (optionalRecords.isPresent()) {
-      populateLoadResult(uuids, optionalRecords.get(), srsLoadResult);
+      populateLoadResultFromSRS(uuids, optionalRecords.get(), srsLoadResult);
     } else {
       srsLoadResult.setInstanceIdsWithoutSrs(uuids);
     }
     return srsLoadResult;
   }
 
   @Override
-  public List<JsonObject> loadInventoryInstancesBlocking(Collection<String> instanceIds, OkapiConnectionParams params) {
-    return new ArrayList<>();
+  public List<JsonObject> loadInventoryInstancesBlocking(Collection<String> instanceIds, OkapiConnectionParams params, int partitionSize) {
+    Optional<JsonObject> optionalRecords = client.getByIdsFromInventory(new ArrayList<>(instanceIds), params, partitionSize);
+    return optionalRecords.map(this::populateLoadResultFromInventory).orElseGet(ArrayList::new);
```

Comment:
It is possible that the UUIDs are not present in Inventory as well. Until we get to handling logs feature. Let's atleast log that the list of UUIDs not found in inventory too

Reply:
@KVupp should it be the ERROR log level?

Reply:
@andrii-novytskyi    log.info

Reply:
Added log with the number of not found instances to the ExportManager